### PR TITLE
Cleanup code and add more explanatory comments

### PR DESCRIPTION
**Pull request by bianca-stana**
This PR:
- Cleans up most of the code in this project (all of the changes are categorised by commit; thus, the PR is easier to review by commit instead of by file)
- Adds new explanatory comments (6bbfcd8) and rephrases some existing ones (caf32da)
- Adds an app icon (7f4de99)

### CODE REVIEW

File: Assets/Editor/Tools/SRTImporter/SrtImporter.cs [hunk 1]
```diff
@@ -1,31 +1,29 @@
-using UnityEditor.AssetImporters;
 using System.IO;
+using Unity.Samples.ClosedCaptions;
+using UnityEditor.AssetImporters;
 
-namespace Unity.Samples.Accessibility
+namespace Unity.Samples.ClosedCaptionsEditor
```
Comment (juliaperdigueiroUnity) [inline Assets/Editor/Tools/SRTImporter/SrtImporter.cs:5]: I was the one who made the call on the namespaces and asked Alex/Shaz to go with Accessibility with code that you could copy/paste into a project to use (so all the Accessibility related features), and the game-specific code was in the LetterSpell namespace. I see you moved code separately into Accessibility (screen reader code) and Closed Captions (closed captions code).
I'm not against separating, but I also don't want us to make things overcomplicated either. I suggest we decide together during one of our team syncs where we want all the code to sit, and what namespaces we want to use (also the namespace doesn't have to match the folder, so we could still use the same namespace and only separate the code in folders if the main goal is for users to be able to copy parts of the code more easily).

PS: I know this is not the same namespace as the code under the Script folder (this is the Editor stuff) but the idea of deciding all of the namespaces as a team stands so I'll leave the comment here 🙃

**Review by juliaperdigueiroUnity — COMMENTED** [review]
I haven't finished reading through but I won't have time to finish it all tonight and wanted to submit the comment I already made so... here it is 🙂

File: Assets/Editor/Tools/SRTImporter/SrtImporter.cs (hunk 1 above)
Comment (bianca-stana) [reply Assets/Editor/Tools/SRTImporter/SrtImporter.cs:5]: As discussed with the team, I renamed the Accessibility folder and namespace in 93c8bce and fc2c6a3.

File: Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs [hunk 2]
```diff
@@ -42,24 +43,28 @@ void Awake()
         
         void Start()
         {
-            MusicManager.audioPlayingStatusChanged += MusicManagerOnAudioPlayingStatusChanged;
+            AudioManager.audioPlayingStatusChanged += OnAudioPlayingStatusChanged;
```
Comment (alexandredzi) [inline Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs]: This does not survive domain reload unfortunately. Moving it to OnEnable (and OnDisable) would be better.

File: Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs [hunk 3]
```diff
@@ -42,24 +43,28 @@ void Awake()
...
+            m_SubtitleMap.Add(AudioManager.instance.successEffect.name, subtitles[1]);
+            m_SubtitleMap.Add(AudioManager.instance.failureEffect.name, subtitles[2]);
         }
 
         void OnDestroy()
         {
             s_Instance = null;
-            MusicManager.audioPlayingStatusChanged -= MusicManagerOnAudioPlayingStatusChanged;
+
+            AudioManager.audioPlayingStatusChanged -= OnAudioPlayingStatusChanged;
```
Comment (alexandredzi) [inline Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs:60]: See previous comments

File: Assets/Scripts/Closed Captions/UnityTimeSpan.cs [hunk 4]
```diff
@@ -1,43 +1,42 @@
...
     /// <summary>
     /// This class represents a time span in milliseconds.
     /// It is used to represent the start and end time of a subtitle item.
     /// This class is serializable so that it can be used in the inspector.
     /// </summary>
+    [Serializable]
     public struct UnityTimeSpan : IEquatable<UnityTimeSpan>
     {
         [SerializeField]
-        private int m_Milliseconds;
```
Comment (alexandredzi) [inline Assets/Scripts/Closed Captions/UnityTimeSpan.cs:16]: This should remain 
`int m_Milliseconds`

File: Assets/Scripts/Closed Captions/UnityTimeSpan.cs [hunk 5]
```diff
@@ -1,43 +1,42 @@
...
+        int milliseconds;
         
         public int Milliseconds
```
Comment (alexandredzi) [inline Assets/Scripts/Closed Captions/UnityTimeSpan.cs]: Should be  
`public int milliseconds`

**Review by alexandredzi — APPROVED** [review]
minor comments

File: Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs (hunk 2 above)
Comment (bianca-stana) [reply Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs]: Fixed in 838ac55.

File: Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs (hunk 3 above)
Comment (bianca-stana) [reply Assets/Scripts/Closed Captions/ClosedCaptionsManager.cs:60]: Fixed in 838ac55.

File: Assets/Scripts/Closed Captions/UnityTimeSpan.cs (hunk 5 above)
Comment (bianca-stana) [reply Assets/Scripts/Closed Captions/UnityTimeSpan.cs]: Fixed in 838ac55.

File: Assets/Scripts/Closed Captions/UnityTimeSpan.cs (hunk 4 above)
Comment (bianca-stana) [reply Assets/Scripts/Closed Captions/UnityTimeSpan.cs:16]: Fixed in 838ac55.

**Pull request merged**